docs/user-guide/src/OWNERS

```diff
@@ -31,4 +30,5 @@ emeritus_approvers:
 
 emeritus_reviewers:
  - macaptain
- - namnx228
\ No newline at end of file
+ - namnx228
+ - dukov
```

Comment:
Please move this line before macaptain to follow the alphabetical order

Reply:
Done